### ISSUE TITLE
Is prepack required?

### ISSUE THREAD

**Issue by danedmunds** [nanogiants/nestjs-swagger-api-exception-decorator#60]
I've noticed that prepack seems to be bringing in quite a few security issues

```
❯ npm audit  
# npm audit report

jszip  <=3.7.1
Severity: high
Prototype Pollution - https://github.com/advisories/GHSA-jg8v-48h5-wgxg
JSZip contains Path Traversal via loadAsync - https://github.com/advisories/GHSA-36fh-84j7-cv5h
fix available via `npm audit fix --force`
Will install prepack@0.2.24, which is a breaking change
node_modules/jszip
  node-zip  >=1.1.0
  Depends on vulnerable versions of jszip
  node_modules/node-zip
    prepack  >=0.2.25
    Depends on vulnerable versions of fbjs
    Depends on vulnerable versions of node-zip
    Depends on vulnerable versions of zip-dir
    node_modules/prepack
  zip-dir  1.0.0 - 1.0.2
  Depends on vulnerable versions of jszip
  node_modules/zip-dir

node-fetch  <=2.6.6
Severity: high
The `size` option isn't honored after following a redirect in node-fetch - https://github.com/advisories/GHSA-w7rc-rwvf-8q5r
node-fetch forwards secure headers to untrusted sites - https://github.com/advisories/GHSA-r683-j2x4-v87g
fix available via `npm audit fix --force`
Will install prepack@0.2.24, which is a breaking change
node_modules/isomorphic-fetch/node_modules/node-fetch
  isomorphic-fetch  2.0.0 - 2.2.1
  Depends on vulnerable versions of node-fetch
  node_modules/isomorphic-fetch
    fbjs  0.7.0 - 1.0.0
    Depends on vulnerable versions of isomorphic-fetch
    node_modules/fbjs

7 high severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force
```

Is this dependency required? I can't find where it's being used and the tests pass when I remove it. Unless prepack somehow passively optimize code?
It seems like the last release for prepack ended was in 2018 and the repo has been archived since Feb 2022.

**Comment by dasheck0** [comment]
Hi @danedmunds Thanks for notifying us about this. It is not needed. We added this to cope with recent security issues and it fixed some of them at the time. We ran a clean install without prepack which resulted in 0 vulnerabilities. So it is most definitely not needed anymore. Updates are available with version 1.6.11 on npm and github releases. If anything comes up please let me know! :)

**Comment by danedmunds** [comment]
Thanks so much @dasheck0 !
